t/perf/opcount.t

```diff
@@ -712,4 +712,11 @@ test_opcount(0, "builtin::isbool gets constant-folded",
                     const    => 1,
                 });
 
+test_opcount(0, "builtin::weaken is replaced with direct opcode",
```

Comment:
Should we test for `unweaken` and `isweak` too, just for completeness?

Reply:
Yeah; why not? Added 37622de526